store/datastore.go

```diff
@@ -13,15 +15,26 @@ type DataStore struct {
 }
 
 //go:generate moq -out datastoretest/datastore.go -pkg storetest . Storer
+//go:generate moq -out datastoretest/mongo.go -pkg storetest . MongoDB
 
 // Storer represents basic data access via Get, Remove and Upsert methods (to be implemented in the future).
-type Storer interface {
+type dataMongoDB interface {
```

Comment:
Is there a reason this is now private?

Reply:
I've changed this back... and removed the other Storer interface. It made no sense.